blog/2025/pcsx2-2.6/index.mdx

```diff
@@ -0,0 +1,284 @@
+</PCSX2PRLink>
+
+SternXD has been on a mission to achieve feature parity between Big Picture Mode and the Qt interface. Thanks to his work, you can now adjust Network and HDD Settings, Create a Memory Card as well as login to RetroAchievements directly from Big Picture Mode!
+
+<Image src={require("./img/BPM_network.webp").default} />
+
+<PCSX2PRLink authors="TheTechnician27" prNums="13199">
+  ### Korean Game Titles
+</PCSX2PRLink>
+
+TheTechnician27 has been working on adding Korean game titles to most NTSC-K serials which did not previously have them. PCSX2 now has a 99% complete Korean translation on Crowdin (at the time of writing), so this has been a long time coming. Despite NTSC-K having nearly 700 known serials, the GameDB for a long time has only had Latin versions of the titles, either pulled from the NTSC-U/PAL name or from a Romaji version of the Japanese name.
+
+<PCSX2PRLink authors="kamfretoz" prNums="12746">
+  ### Game Shortcut Creation Feature
+</PCSX2PRLink>
+
+Motivated by [this GitHub issue](https://github.com/PCSX2/pcsx2/issues/8349), KamFretoZ has set his sights on implementing a feature where you can automatically create a desktop (or start menu) shortcut for your game right from PCSX2 so you can launch your games right from your desktop! This one has been requested a lot and we've seen quite a few people ask about it on our Discord server.
+
+Only available for Windows and Linux.
+
+<Image src={require("./img/shortcut_dialog.webp").default} />
+
+<PCSX2PRLink authors="kamfretoz,SternXD" prNums="12722,13410">
+  ### Custom Background Support
+</PCSX2PRLink>
+
+What started as a silly idea between KamFretoZ and JordanTheToast to see what's possible with Qt, ended up becoming a full blown feature. With this, you can now set a custom background on your game list to your heart's content.
+
+<Image src={require("./img/custom_background.webp").default} />
+
+<Image src={require("./img/BPM_Background.webp").default} />
+
+<PCSX2PRLink authors="TheLastRar" prNums="13009,12678,12970">
+  ### Emoji Support for Big Picture Mode
+</PCSX2PRLink>
+
+Allows emojis to be shown in the Big Picture Mode, the fullscreen pause menu and any rendered
+RetroAchievements popups would often use emojis, and we are now able to render these.
+
+Not all emojis are supported, that is a limitation of the library (ImGui) we use for this.
+
+For the longest time our Big Picture Mode UI implemented using [Dear ImGui](https://github.com/ocornut/imgui) was unable to display emoji due to upstream limitations (Especially more apparent when displaying emojis from certain RetroAchievements sets). Thanks to [upstream fixes](https://github.com/ocornut/imgui/issues/8465) and TheLastRar's efforts, we can finally go ahead and implement emoji supports right on PCSX2's Big Picture Mode.
+
+For technical detail, previous versions of ImGui required us to provide ranges of characters we needed for ImGui to generate a font atlas.
```

Comment:
Consider combining this with the other lines to make a paragraph, rather than multiple disjointed sentences. A few tweaks as well:

"On a more technical level, previous version of ImGui required us to provide ranges of characters we needed. ImGui would then generate a font atlas from this. This required that we either hardcode, or generate these ranges for any languages we support in PCSX2. With ImGui 1.9.21, we can now rely on ImGui to dynamically adjust its font atlas on its own, removing the maintenance burden of fixed character ranges."